frontend/webpack.config.js

```diff
@@ -1,6 +1,13 @@
 // shared config (dev and prod)
 const { resolve } = require("path");
 const HtmlWebpackPlugin = require("html-webpack-plugin");
+const dotenvExpand = require("dotenv-expand");
+const dotenv = require("dotenv");
+const webpack = require("webpack");
+
+const myEnv = dotenv.config();
+dotenvExpand(myEnv);
+
```

Comment:
```suggestion
const dotenv = require("dotenv");
const webpack = require("webpack");
```

Reply:
dotenv-expand package is there do I need to remove this one.. as it is not doing anything.. possibly we can pick this one in another issue ?

Reply:
You can remove this package and update the code in the same PR.

Reply:
@jyash97 done